test/helpers/name.go

```diff
@@ -49,7 +49,11 @@ func ObjectPrefixForTest(t test.T) string {
 
 // ObjectNameForTest generates a random object name based on the test name.
 func ObjectNameForTest(t test.T) string {
-	return AppendRandomString(ObjectPrefixForTest(t))
+	nameprefix := ObjectPrefixForTest(t)
```

Comment:
Can we use kmeta.ChildName here?

Reply:
Yes, `ChildName` should be used. Updated. Thank you.